routing-ortools/main.py

```diff
@@ -210,45 +196,35 @@ def capacity_callback(from_index):
             max_stops_in_vehicle = max(max_stops_in_vehicle, stop_count)
             min_stops_in_vehicle = min(min_stops_in_vehicle, stop_count)
 
-        # Creates the statistics.
-        statistics = {
-            "result": {
-                "custom": {
+        statistics = nextmv.Statistics(
+            run=nextmv.RunStatistics(duration=end_time - start_time),
+            result=nextmv.ResultStatistics(
+                value=solution.ObjectiveValue(),
+                custom={
                     "solution_found": True,
                     "activated_vehicles": activated_vehicles,
                     "max_route_duration": max_route_duration,
                     "max_stops_in_vehicle": max_stops_in_vehicle,
                     "min_stops_in_vehicle": min_stops_in_vehicle,
                 },
-                "duration": end_time - start_time,
-                "value": solution.ObjectiveValue(),
-            },
-            "run": {
-                "duration": end_time - start_time,
-            },
-            "schema": "v1",
-        }
+            ),
+        )
     else:
-        routes = []
-        statistics = {
-            "result": {
-                "custom": {
+        statistics = nextmv.Statistics(
+            run=nextmv.RunStatistics(duration=end_time - start_time),
+            result=nextmv.ResultStatistics(
+                value=None,
+                custom={
                     "solution_found": False,
                 },
-                "duration": end_time - start_time,
-                "value": None,
-            },
-            "run": {
-                "duration": end_time - start_time,
-            },
-            "schema": "v1",
-        }
-
-    return {
-        "solutions": [{"vehicles": routes, "unplanned": []}],
-        "statistics": statistics,
-        "version": {"ortools": ortools_version},
-    }
+            ),
+        )
+
+    return nextmv.Output(
+        options=options,
+        solution={"vehicles": routes, "unplanned": []},
```

Comment:
While I do agree with _not_ continuing to always return a list of solutions, this is breaking the expectations of the frontend route visualization (and immediate comparability to nextroute output).
Should we avoid the breaking change in the output? (for `routing-*` in particular) Or, should we adapt to it? Are these starter apps used by anyone in marketplace directly? :thinking: